### PR TITLE
data:saikumarb12345

### PR DESCRIPTION
**Pull request by saikumarb12345**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6174"></a>

### CODE REVIEW

File: data/saikumarb12345.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Boyapati Saikumar",
+  "type": "personal",
+  "bio": "I am Open Source enthusiast and Learning frontend development",
+  "avatar":"https://github.com/saikumarb12345.png",
```
Comment (khushalarora) [inline data/saikumarb12345.json:5]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @saikumarb12345, thank you for contributing :tada:

**Comment by saikumarb12345** [comment]
updated line 5

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.eddiehub.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**